parlai/crowdsourcing/tasks/turn_annotations_static/analysis/compile_results.py

```diff
@@ -232,9 +236,21 @@ def compile_initial_results(self, results_folders) -> list:
                 continue
 
             # HIT-level metric of HIT completion time has to be done here for now
-            task_completion_time_seconds = (
-                data['times']['task_end'] - data['times']['task_start']
-            )
+            try:
+                task_completion_time_seconds = (
+                    data['times']['task_end'] - data['times']['task_start']
+                )
+            except KeyError:
+                # We've been relying on non-mephisto API, and in 1.0.2 'times' was deprecated
```

Comment:
Hmm - doing a simple search, it looks like `parlai/crowdsourcing/tasks/acute_eval/acute_eval_agent_state.py` and `parlai/crowdsourcing/tasks/dialcrowd/webapp-results/src/components/requesters/template/category/CategoryQuality.js` use this `'times'` key as well. I assume these will also need to be ported over? Or is the change in this file only a result of the fact that turn_annotations_static is still not using DataBrowser?

Reply:
Good callout! I missed the `AcuteEvalAgentState`. We can drop the `times` reference here as Mephisto now does this under-the-hood as part of the `AgentState.metadata` object.

For `results/src/components/requesters/template/category/CategoryQuality.js` (and the `dialcrowd` code in general) I was having a hard time understanding what is and is not important to change. As it stands, I'm pretty sure it doesn't function quite correctly as the version of Mephisto their frontend uses predates `v1.0.0`. 

In the interest of not breaking anything _more_ though I'll add a fix for this too.

Reply:
Okay, great, thanks! Yeah, in the absence of frontend testing, it won't be easy to tell how much `dialcrowd` is becoming out of date, especially since we have no in-house maintainer of this code. @klshuster happy to hear your thoughts on this as well - maybe we should just mark this task as working on Mephisto v1.0.0 and untested on later versions?

Also @JackUrb it looks like there's still a lint error?

Reply:
Lint error doesn't appear locally, or two commits ago, and is on a file this PR doesn't touch. Not sure how to resolve (should I change that file myself?)

Reply:
ha good catch :/ Okay, I'm approving this now just because that shouldn't block you